### DISCUSSION TITLE
command not found: raw error

### DISCUSSION THREAD

**Discussion by gimdongwon** [vercel/turborepo#7060] [Help] [answered]
I met this error. plz help me from this error

I input this command

`turbo gen workspace --copy`

![스크린샷 2024-01-22 10 48 25](https://github.com/vercel/turbo/assets/37524650/febdeead-ffb2-4fb0-91e3-d393e6a2c429)

**Comment by anthonyshew** [comment, marked as answer]
See #7061.
